examples/angular-cli/src/stories/basics/component-with-complex-selectors/attribute-selector.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'storybook-attribute-selector[foo]',
+  template: '<p>Attribute selector</p>',
```

Comment:
I find these examples very interesting. 

We could display the html output expected in template 🤔 like :
```ts
template: `
<h3>Attribute selector</h3>
Selector: "storybook-attribute-selector[foo]" <br>
Generated tempalte: "<storybook-attribute-selector foo></storybook-attribute-selector>"
`,
```

but maybe it's too much 🤷‍♂️

Reply:
Done